ttnn/cpp/ttnn/operations/experimental/ccl/all_gather_matmul/device/all_gather_matmul_op.cpp

```diff
@@ -33,7 +34,7 @@ void AllGatherMatmul::validate(
     this->all_gather_struct.validate({input_tensor});
 
     // Matmul validate.
-    this->matmul_struct.validate({all_gather_output_tensor, weight_tensor}, optional_input_tensors);
+    this->matmul_struct.validate({all_gather_output_tensor, weight_tensor}, optional_input_tensors, optional_output_tensors);
```

Comment:
Does it work to just pass in `{}` and not change the parameters of `AllGatherMatmul::validate`? That would be preferable.

Reply:
We'll work on each op in it's own PR.